internal/proxy/meta_cache.go

```diff
@@ -186,6 +189,71 @@ func (m *MetaCache) GetCollectionSchema(ctx context.Context, collectionName stri
 	return collInfo.schema, nil
 }
 
+func (m *MetaCache) IsCollectionLoaded(ctx context.Context, msgID UniqueID, collectionName string) error {
+	m.mu.RLock()
+	collInfo, ok := m.collInfo[collectionName]
+	if !ok {
+		m.mu.RUnlock()
+		return fmt.Errorf("cannot find collection info in MetaCache, collectionName = %s", collectionName)
```

Comment:
What if collection not in cache but in meta?

Reply:
fixed